src/meta/app/src/schema/auto_increment.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use kvapi_impl::AutoIncrementRsc;
+
+use crate::principal::AutoIncrementKey;
+use crate::schema::AutoIncrementStorageIdent;
+use crate::schema::SequenceMeta;
+use crate::tenant_key::ident::TIdent;
+use crate::KeyWithTenant;
+
+/// AutoIncrementIdent is the Ident of Sequence in AutoIncrement,
+/// which is used to distinguish SequenceIdent and manage resource recycling and show meta separately.
+pub type AutoIncrementIdent = TIdent<AutoIncrementRsc, AutoIncrementKey>;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct AutoIncrementMeta {
+    pub step: i64,
+    pub current: u64,
```

Comment:
If not store storage_version how to split old and new version when we support storage_version=1 in future?